service/cloudconfigv3/master_template.go

```diff
@@ -0,0 +1,275 @@
+package cloudconfigv3
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/certificatetpr"
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_3_0_0"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/randomkeytpr"
+)
+
+// NewMasterTemplate generates a new master cloud config template and returns it
+// as a base64 encoded string.
+func (c *CloudConfig) NewMasterTemplate(customObject v1alpha1.AWSConfig, certs certificatetpr.CompactTLSAssets, keys randomkeytpr.CompactRandomKeyAssets) (string, error) {
+	var err error
+
+	var params k8scloudconfig.Params
+	{
+		params.Cluster = customObject.Spec.Cluster
+		params.EtcdPort = customObject.Spec.Cluster.Etcd.Port
```

Comment:
Here we use v3 of k8scloudconfig and set the port to 2379 not 443. This lets us remove the ELB.